server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -75,8 +79,8 @@ export class TenantManager {
     /**
      * Retrieves the details for the given tenant
      */
-    public async getTenant(tenantId: string): Promise<ITenantConfig> {
-        const tenant = await this.getTenantDocument(tenantId);
+    public async getTenant(tenantId: string, includedDeleted = false): Promise<ITenantConfig> {
+        const tenant = await this.getTenantDocument(tenantId, includedDeleted);
```

Comment:
Just want to call out that this is a good security feature as well. This way, we don't have to manually query the database to check if the tenant is disabled or not. :)